recipes/_rsync.rb

```diff
@@ -30,10 +30,13 @@
 
 include_recipe 'omnibus::_compile'
 
```

Comment:
As @scotthain suggested, let's just skip this recipe on PPC. WE can then remove all the modifications to the `remote_install` resource.